Marlin/src/libs/W25Qxx.cpp

```diff
@@ -53,19 +53,9 @@ void W25QXXFlash::init(uint8_t spiRate) {
 
   OUT_WRITE(SPI_FLASH_CS_PIN, HIGH);
 
-  /**
-   * STM32F1 APB2 = 72MHz, APB1 = 36MHz, max SPI speed of this MCU if 18Mhz
-   * STM32F1 has 3 SPI ports, SPI1 in APB2, SPI2/SPI3 in APB1
-   * so the minimum prescale of SPI1 is DIV4, SPI2/SPI3 is DIV2
-   */
-  #if SPI_DEVICE == 1
-    #define SPI_CLOCK_MAX SPI_CLOCK_DIV4
-  #else
-    #define SPI_CLOCK_MAX SPI_CLOCK_DIV2
-  #endif
   uint8_t clock;
   switch (spiRate) {
-    case SPI_FULL_SPEED:    clock = SPI_CLOCK_MAX;  break;
```

Comment:
This change seems like it does not belong with this PR. @rhapsodyv do you know anything about this?

Reply:
Sure. In fact we can just keep as it is today. I'm changing those spi calls anyway.

Reply:
Keep it as it is _here_ today? Or keep it as it is _in bugfix_ today, before these changes?